packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -434,10 +442,17 @@ export abstract class ApplicationLoadBalancedServiceBase extends Construct {
 
     const internetFacing = props.publicLoadBalancer ?? true;
 
+    if (props.idleTimeout) {
+      if (props.idleTimeout > Duration.seconds(4000)) {
+        throw new Error( 'IdleTime cannot exceed 4000 seconds');
```

Comment:
Looks like the minimum is also 1 so we should check against that as well. We don't want to allow for a number below 1 either.

Reply:
Ah yeah totally agree. Added that guardrail in